mcapi/src/v1_16_5/java/net/flintmc/mcapi/v1_16_5/chat/component/event/VersionedClipboardActionExecutor.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * FlintMC
+ * Copyright (C) 2020-2021 LabyMedia GmbH and contributors
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package net.flintmc.mcapi.v1_16_5.chat.component.event;
+
+import com.google.inject.Singleton;
+import net.flintmc.mcapi.chat.component.event.ClickActionExecutor;
+import net.flintmc.mcapi.chat.component.event.ClickEvent;
+import net.flintmc.mcapi.chat.component.event.ClickEvent.Action;
+import net.flintmc.mcapi.chat.component.event.ClickEventAction;
+import net.minecraft.client.Minecraft;
+
+@Singleton
+@ClickEventAction(value = Action.COPY_TO_CLIPBOARD, priority = -1)
```

Comment:
Is there a reason `priority` is `-1`? The default priority seems to be `1`, so `0` should be fine for this this event